### PR TITLE
Linked Account Data

### LINKED ISSUE #27: Query account details
An account can be retrieved by the following query

```graphql
query account{
	account(address:"0xAe1ec841923811219b98ACeB1db297AADE2F46F3") {
    balance
    totalBalance{
      lockedGold
    }
    accountSummary
    isAccount
    isSigner
    lockedGold
    attestation
  }
}
```

Here is the sample result

```json
{
  "data": {
    "account": {
      "balance": 11882463899300000000,
      "totalBalance": {
        "lockedGold": 1e+22
      },
      "accountSummary": {
        "address": "0xAe1ec841923811219b98ACeB1db297AADE2F46F3",
        "name": "CLabs Validator #5 on alfajores",
        "authorizedSigners": {
          "vote": "0xAe1ec841923811219b98ACeB1db297AADE2F46F3",
          "validator": "0xAe1ec841923811219b98ACeB1db297AADE2F46F3",
          "attestation": "0xb41697131c189F5e74Cf7A065D8CaA625F26EdE5"
        },
        "metadataURL": "https://storage.googleapis.com/clabs_validator_metadata/alfajores/validator-alfajores-0xAe1ec841923811219b98ACeB1db297AADE2F46F3-metadata.json",
        "wallet": "0x0000000000000000000000000000000000000000",
        "dataEncryptionKey": "0xe5688467b94fb8c5b3bf2c2149ecf1530c2f9c16d5347b62acc1f3fd3ff1d7fa094f8194e4b517e959caf9cc4d54272f67e1612cfdd71424bbab5be467d6a42a"
      },
      "isAccount": true,
      "isSigner": false,
      "lockedGold": {
        "total": {
          "s": 1,
          "e": 22,
          "c": [
            100000000
          ]
        },
        "nonvoting": {
          "s": 1,
          "e": 22,
          "c": [
            100000000
          ]
        },
        "requirement": {
          "s": 1,
          "e": 22,
          "c": [
            100000000
          ]
        },
        "pendingWithdrawals": [],
        "pendingWithdrawalsTotal": {
          "s": 1,
          "e": 0,
          "c": [
            0
          ]
        }
      },
      "attestation": {
        "requestFees": 0,
        "expiryBlocks": 720
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by MonikaCat**
Fixes #27

### CODE REVIEW

File: components/account/AccountList.tsx [hunk 1]
```diff
@@ -102,14 +105,14 @@ const useStyles = makeStyles(({ spacing }) => {
 
 
 const AccountList = () => {
-  const rowsOption1 = 10;
-  const rowsOption2 = 30;
-  const rowsOption3 = 50;
+  const rowsOption1 = 20;
```
Comment (kwunyeung) [inline components/account/AccountList.tsx:108]: @MonikaCat better put the parameter in the env variables

File: components/account/AccountList.tsx [hunk 2]
```diff
@@ -102,14 +105,14 @@ const useStyles = makeStyles(({ spacing }) => {
...
+  const rowsOption2 = 50;
+  const rowsOption3 = 100;
 
   const classes = useStyles();
 
   const [page, setPage] = React.useState(1);
-  const [pageSize, setPageSize] = React.useState(10)
+  const [pageSize, setPageSize] = React.useState(20)
```
Comment (kwunyeung) [inline components/account/AccountList.tsx:115]: The default value should be from the options

File: components/account/AccountOverview.tsx [hunk 3]
```diff
@@ -38,6 +37,9 @@ import ComponentLoader from '../misc/ComponentLoader';
 import NotAvailable from '../misc/NotAvailable'
 import ErrorMessage from '../misc/ErrorMessage';
 import MiddleEllipsis from '../misc/MiddleEllipsis'
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+
+const BigNumber = require('bignumber.js');
```
Comment (kwunyeung) [inline components/account/AccountOverview.tsx:42]: Import as ES module instead of CommonJS

File: components/account/AccountPage.tsx [hunk 4]
```diff
@@ -56,35 +56,35 @@ const AccountPage = (props: any) => {
...
 
-      <Grid item xs={12}  >
+      {/* <Grid item xs={12} >
         <CoinBalanceHistory />
-      </Grid>
+      </Grid> */}
 
       <Grid item xs={12} >
-        <Downtime />
+        <Downtime address={accountAddress} />
```
Comment (kwunyeung) [inline components/account/AccountPage.tsx:78]: Hide `Downtime` and `ProposedBlocks` if the address is not a validator

File: components/account/Downtime.tsx [hunk 5]
```diff
@@ -102,22 +61,25 @@ const useStyles = makeStyles(({ spacing }) => {
...
   }
 });
 
 
+type DowntimeProps = { address: string };
 
-
-const Downtime = () => {
+const Downtime = ({ address }: DowntimeProps) => {
   const rowsOption1 = 10;
```
Comment (kwunyeung) [inline components/account/Downtime.tsx:83]: Same as above

File: components/account/AccountOverview.tsx [hunk 6]
```diff
@@ -266,39 +290,41 @@ const AccountOverview = (props: any) => {
...
+              )} CELO
+            </Typography> : < NotAvailable variant="body2" className={classes.alignRight} />}
           </Grid>
-          <Grid item xs={12} md={12} >
+          <Grid item xs={12}  >
             {accountQuery.data.account && accountQuery.data.account.balance && chainQuery.data.chain && chainQuery.data.chain.tokenPrice && chainQuery.data.chain.tokenPrice.usd ?
               < Typography variant="body2" className={classes.alignRight} noWrap>
-                $ {numbro((accountQuery.data.account.balance * chainQuery.data.chain.tokenPrice.usd).toLocaleString('fullwide',)).format("0.00")}
+                ${BigNumber.prototype.toFormat.call(
+                new BigNumber(accountQuery.data.account.balance * chainQuery.data.chain.tokenPrice.usd)
```
[the remark this answers is not in the document]
Comment (MonikaCat) [reply components/account/AccountOverview.tsx]: Ok, originally, we had it showing as $ in our design :)  I have updated it to cUSD :)

File: components/account/AccountOverview.tsx [hunk 7]
```diff
@@ -314,14 +340,14 @@ const AccountOverview = (props: any) => {
 
           <Grid item xs={8} md={8}> {accountQuery.data.account && accountQuery.data.account.totalBalance && accountQuery.data.account.totalBalance.gold && accountQuery.data.account.totalBalance.usd ?
```
[the remark this answers is not in the document]
Comment (MonikaCat) [reply components/account/AccountOverview.tsx:341]: Hmm this is per our design, however, I don't feel like there is a need for it atm.  Currently, the only function of this dropdown is to display the balance of the user, which, as you mentioned, is already shown above. I can remove it if you suggest it.

File: components/account/Transactions.tsx [hunk 8]
```diff
@@ -130,75 +127,107 @@ const AccountTransactions = () => {
...
-                              </Link>
-                            </Typography>
-                          </Grid>
-                          <Grid item xs={4} >
-                            <Typography variant="body2" className={classes.alignRight}>
-                              {row.time}
-                            </Typography>
-                          </Grid>
+          <Grid item xs={12}>
+            <TableContainer className={classes.container}>
```
[the remark this answers is not in the document]
Comment (MonikaCat) [reply components/account/Transactions.tsx:143]: Ok, I have created a new issue for this, link here:  https://github.com/forbole/big-dipper-celo-client/issues/61

File: components/account/AccountOverview.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (MonikaCat) [reply components/account/AccountOverview.tsx]: I have removed Token Dropdown and updated the code :)

**Pull request merged**